src/extractor/guidance/intersection_handler.cpp

```diff
@@ -383,136 +393,140 @@ std::size_t IntersectionHandler::findObviousTurn(const EdgeID via_edge,
     if (intersection.size() == 2)
         return 1;
```

Comment:
wasn't the plan to substitute these angle and `0` is `no obvious candidate` by `boost::optional`?